docs/04-Contributing.md

```diff
@@ -14,6 +14,14 @@ Welcome to this Beginner-friendly instructional guide for individual contributor
 
 > Guess what! If you are already a member of 100Devs, one of us can personally walk you through all the steps you will need to contribute code to this project. Please send a message on Discord in the [Together discussion thread (external link)](https://discord.com/channels/735923219315425401/1038482732633825442). We will help you no matter what stage of learning you are on with whatever we can. We can assist you in looking for an issue to address, and if no issue exists that fits, we will create one. 🤠
 
+
+Contributing Guidelines
```

Comment:
Note Devlins comments above as well.
For Markdown if you add two hashes ('#') before the subtitle it will become easier to find.